sharding-proxy/sharding-proxy-bootstrap/src/main/resources/conf/server.yaml

```diff
@@ -29,11 +29,6 @@
 #    namespace: orchestration
 #    props:
 #      overwrite: false
-#      retry-interval-milliseconds: 10
-#      time-to-live-seconds: 10
-#      max-retries: 10
-#      operation-timeout-milliseconds: 10
-#      digest: 123
 #
```

Comment:
If we didn't remove these parameters from config file, I suggest keep them here for examples.

Reply:
I have reverted the config.